src/main/java/competition/subsystems/vision/VisionSubsystem.java

```diff
@@ -131,7 +134,9 @@ public Optional<EstimatedRobotPose> getPhotonVisionEstimatedPose(Pose2d previous
             //return customPhotonPoseEstimator.update();
             photonPoseEstimator.setReferencePose(previousEstimatedRobotPose);
             var estimatedPose = photonPoseEstimator.update();
-            if (!estimatedPose.isEmpty() && this.isEstimatedPoseReliable(estimatedPose.get())) {
+            var isReliable = isEstimatedPoseReliable(estimatedPose.get());
```

Comment:
the copy paste of all this code for front + rear is going to bite us some day soon, but maybe the season will end around then 🤞🏻